src/winetricks

```diff
@@ -2341,6 +2344,28 @@ _EOF_
 
 #---- Private Functions ----
 
+winetricks_dl_warning() {
+    case $LANG in
+    ru*) _W_countrymsg="Скрипт определил, что ваш IP адрес принадлежит России. Если во время загрузки файлов вы увидите ошибки несоответствия сертификата, скачайте файлы вручную или используйте VPN."
+      ;;
+    *)  _W_countrymsg="Your IP address has been determined to belong to Russia. If you encounter a certificate error while downloading, please use VPN or download files manually."
+      ;;
+    esac
+
+    # Lookup own country via IP address only once (i.e. don't run this function for every download invocation)
+    if [ -z "$MYCOUNTRY" -a -x "`which geoiplookup 2>/dev/null`" ]
+    then
+        ip_address=`wget http://ipinfo.io/ip -q -O -`
+        export MYCOUNTRY=`geoiplookup "$ip_address" | awk '/Country Edition/{print $4}' | sed 's/,//'`
+        if [ "$MYCOUNTRY" = "RU" ]
+        then
+           w_warn "$_W_countrymsg"
+        fi
+    else
+        export MYCOUNTRY="Unknown"
```

Comment:
If you're going to name this variable country, it should support all countries, not just Russia. Using the ipinfo solution above also solves this..

That could then be used for anything else country specific that may be needed in the future.